enigma-core/app/src/db/dal.rs

```diff
@@ -51,9 +51,14 @@ impl DB {
             Ok(list) => list,
             Err(_) => Vec::new(),
         };
-        // converts the Strings to slices (str)
-        let cf_list_burrowed = cf_list.iter().map(String::as_str).collect::<Vec<&str>>();
-        let database = rocks_db::open_cf(&options, &location, &cf_list_burrowed[..])?;
+        // converts the Strings to descriptors (adds to each cf an options object)
+        let cf_descriptors = cf_list.into_iter().map(|name| {
+            let prefix_extractor = SliceTransform::create_fixed_prefix(1);
```

Comment:
It is better to use `DELTA_PREFIX` here and in line 42 instead of a hard-coded value `1`.